### PR TITLE
xxhash v0.8.0

### PR DESCRIPTION
**Pull request by hros**

### CODE REVIEW

File: X/xxHash/build_tarballs.jl [hunk 1]
```diff
@@ -3,28 +3,22 @@
...
 CPPFLAGS=-DXXH_INLINE_ALL make -j${nproc}
 make install
-
-if [[ "${target}" == *-mingw* ]]; then
-    cd "${prefix}/lib"
-    ar x libxxhash.dll.a
-    cc -shared -o "${libdir}/libxxhash.dll" *.o
-    rm *.o
-fi
+if [[ "${target}" == *-mingw* ]]; then     cd "${prefix}/lib";     ar x libxxhash.dll.a;     cc -shared -o "${libdir}/libxxhash.dll" *.o;     rm *.o; fi
```
Comment (giordano) [inline X/xxHash/build_tarballs.jl:21]: Why did you change these lines? This makes reading the command much harder

Comment (hros) [reply X/xxHash/build_tarballs.jl]: non intentional. caused from copy pasting in the terminal.
updated to original multi-line version with another commit
